scripts/run_celery_exit.sh

```diff
@@ -48,8 +48,6 @@ function ensure_celery_is_running {
 
 
 function on_exit {
-  apk add --no-cache procps
-  apk add --no-cache coreutils
```

Comment:
These are the two packages that we want to get installed earlier during Docker image creation.